Index: org.eclipse.linuxtools.rpm.ui.editor/src/org/eclipse/linuxtools/rpm/ui/editor/preferences/PreferenceConstants.java
===================================================================
--- org.eclipse.linuxtools.rpm.ui.editor/src/org/eclipse/linuxtools/rpm/ui/editor/preferences/PreferenceConstants.java	(revision 25288)
+++ org.eclipse.linuxtools.rpm.ui.editor/src/org/eclipse/linuxtools/rpm/ui/editor/preferences/PreferenceConstants.java	(revision 25289)
@@ -70,7 +70,7 @@
 	 */
 
 	public static final String DP_RPM_LIST_FILEPATH = System
-			.getProperty("user.dir") + "/.pkglist"; //$NON-NLS-1$ //$NON-NLS-2$
+			.getProperty("user.home") + System.getProperty("file.separator") + ".pkglist"; //$NON-NLS-1$ //$NON-NLS-2$
 	public static final String DP_RPM_LIST_MAX_PROPOSALS = "10"; //$NON-NLS-1$
 	public static final String DP_RPMTOOLS_RPM = "rpm -qa --qf '%{NAME}\\n'"; //$NON-NLS-1$
 	public static final String DP_RPMTOOLS_YUM = "yum -C list all | grep - | cut -d \".\" -f 1"; //$NON-NLS-1$
